lib/sarif/cargo_audit_sarif.rb

```diff
@@ -4,10 +4,11 @@ class CargoAuditSarif < BaseSarif
 
     CARGO_AUDIT_URI = 'https://github.com/RustSec/cargo-audit/'.freeze
 
-    def initialize(scan_report, repo_path = nil)
+    def initialize(scan_report, repo_path = nil, scanner_config = {})
       super(scan_report, {}, repo_path)
       @uri = CARGO_AUDIT_URI
       @logs = parse_scan_report!
+      @scanner_config = scanner_config
```

Comment:
```ruby
@scanner_config = scanner_config
``` 
appears in multiple sarif adapters.  Is there a way to set it in `super()` instead?
If I'm not mistaken, `super()` is called from every adapter.

Reply:
When generating SARIF reports, it invokes the scanner sarif adapter which then populates the base sarif.